document-generator-accounts/src/main/java/uk/gov/companieshouse/document/generator/accounts/mapper/smallfull/model/ixbrl/balancesheet/BalanceSheet.java

```diff
@@ -0,0 +1,91 @@
+package uk.gov.companieshouse.document.generator.accounts.mapper.smallfull.model.ixbrl.balancesheet;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.gson.Gson;
+import uk.gov.companieshouse.document.generator.accounts.mapper.smallfull.model.ixbrl.balancesheet.capitalandreserves.CapitalAndReserve;
+import uk.gov.companieshouse.document.generator.accounts.mapper.smallfull.model.ixbrl.balancesheet.currentassets.CurrentAssets;
+import uk.gov.companieshouse.document.generator.accounts.mapper.smallfull.model.ixbrl.balancesheet.fixedassets.FixedAssets;
+import uk.gov.companieshouse.document.generator.accounts.mapper.smallfull.model.ixbrl.balancesheet.otherliabilitiesandassets.OtherLiabilitiesAndAssets;
+
+import java.util.Objects;
+
+public class BalanceSheet {
+
+    @JsonProperty("called_up_shared_capital_not_paid")
```

Comment:
This field does not match small-full-accounts tempalte. This should be `called_up_share_capital_not_paid`



called_up_share_capital_not_paid
--


`

Reply:
documentation says "shared"?